github-profiles/rrodriguezca-2238/README.MD

```diff
@@ -0,0 +1,26 @@
+https://github.com/rrodriguezca
+# Robinson Rodríguez CV
+## Description
+I am teacher of English on a quest to learn something new. This new knowledge would be integrated as part of an ongoing effort to become a full stack developer.
+
+## Useful Commants 🤓
+
+Dillinger is currently extended with the following plugins.
+Instructions on how to use them in your own application are linked below.
+
+| Commant | Description |
+| ------ | ------ |
```

Comment:
As Guillermo mentioned, it is better for readability to extend the markdown table width so that all rows have the same width.
It makes it easier to read when looking at the markdown code.

Reply:
Done.